solutions/kubernetes/ceph-binary-build.sh

```diff
@@ -215,7 +215,7 @@ function ceph_build() {
 
                     pushd rpmbuild
                         add_common_separator "Start Build"
-                        rpmbuild --define "_topdir `pwd`" -ba SPECS/ceph.spec
+                        rpmbuild --define "_topdir $(pwd)" -ba SPECS/ceph.spec
```

Comment:
Please validate that script works after this change.

Reply:
As discussed reverted the changes.